### PR TITLE
[OSD-15879] backplane support login to service cluster directly

### PR DESCRIPTION
**Pull request by bmeng**
### What type of PR is this?

_feature_

### What this PR does / Why we need it?
Support login to the service cluster for hypershift clusters directly

### Which Jira/Github issue(s) does this PR fix?

_Resolves #[OSD-15879](https://issues.redhat.com//browse/OSD-15879)_

### Special notes for your reviewer
Tested with the following scenarios:
- Login to service cluster for given hypershift hosted cluster with id/name
- Login to service cluster for given hypershift mgmt_cluster with id/name
- Exit with meaningful error when trying to login to service cluster for a normal osd/rosa cluster

### Pre-checks (if applicable)

- [ ] Ran unit tests locally
- [ ] Validated the changes in a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

File: cmd/ocm-backplane/login/login.go [hunk 1]
```diff
@@ -119,6 +126,18 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 			"Name": clusterName}).Infoln("Management cluster")
 	}
 
+	if args.service {
```
Comment (samanthajayasinghe) [inline cmd/ocm-backplane/login/login.go:129]: It's good to add a unit test to cover the `service`  cluster scenario.
eg : management cluster test scenario : https://github.com/openshift/backplane-cli/blob/3da00fc2a1e6b3392083f7e3d2f0202b6a9cc3ce/cmd/ocm-backplane/login/login_test.go#L214

Comment (bmeng) [reply cmd/ocm-backplane/login/login.go:129]: Thanks, the related testing is added

File: README.md [hunk 2]
```diff
@@ -93,6 +93,15 @@ In this example, we will login to a cluster with id `123456abcdef` in production
   $ oc whoami
   system:serviceaccount:openshift-backplane-srep:1234567
   ```
+  
+- To login to the mgmt cluster for hypershift or the hive cluster for normal OSD/Rosa
+  ```
+  $ ocm backplane login <cluster> --manager
+  ```
+- To login to the service cluster for hypershift hosted cluster or the mgmt cluster
```
Comment (supreeth7) [inline README.md:101]:
```suggestion
- To login to the Service Cluster of a HyperShift hosted cluster or the Management Cluster
```

**Review by supreeth7 — CHANGES_REQUESTED** [review]
Overall looks good, pointed out a few minor nits.
Just tested it, and it works like a charm.

**Comment by bmeng** [comment]
Thanks for the suggestion @supreeth7 fixed the README instruction

**Review by supreeth7 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**